libbeat/docs/index.asciidoc

```diff
@@ -28,6 +28,8 @@ include::./dashboards.asciidoc[]
 
 include::./newbeat.asciidoc[]
 
+include::./config-file-format.asciidoc[]
```

Comment:
conditional stuff in asciidoc? And people complain about me making config files turing complete :grin: